package.json

```diff
@@ -10,7 +10,7 @@
     "dev": "concurrently --kill-others \"npm run start-webpack-dev-server\" \"npm run start-dev-server\"",
     "lint": "eslint -c .eslintrc src",
     "start": "npm run build && npm run start-server",
-    "start-dev-server": "API_HOST=dev-go-api.twreporter.org API_PORT=8080 API_PROTOCOL=http NODE_ENV=development RELEASE_BRANCH=development BABEL_ENV=ssr nodemon ./src/server --exec babel-node",
+    "start-dev-server": "API_HOST=localhost API_PORT=8080 API_PROTOCOL=http NODE_ENV=development RELEASE_BRANCH=development BABEL_ENV=ssr nodemon ./src/server --exec babel-node",
```

Comment:
Currently please use `dev-go-api.twreporter.org`.
After @babygoat docker compose file done, and we can easily start dev API server, we can change it to `localhost`.